docs/src/details/integration/chip_vendors/espressif/overview.rst

```diff
@@ -2,233 +2,73 @@
-   It supports various formats and compression.
-
-   Meanwhile 3rd party libraries
-   (like :ref:`LodePNG<lodepng_rst>` and :ref:`Tiny JPEG<tjpgd>`)
-   allow using image files without conversion.
-
-   After preparing the files, they should be moved to the target device:
-
-   - If properly activated a **SPIFFS** file system based on the ``spiffs_image`` folder should be automatically generated and later flashed to the target
-   - Similar mechanism for **LittleFS** uses the ``flash_data`` folder, but it's only available for Linux hosts
-   - For the **SD Card**, a traditional file browser can be used
-
-- **Invoke proper API calls in the application code**
-
-   The core functionality requires only a few lines. The following example draws the image as well.
-
-   .. code:: c
-
-      #include "esp_spiffs.h"
-
-      void lv_example_image_from_esp_fs(void) {
-
-         esp_vfs_spiffs_conf_t conf = {
-            .base_path = "/spiffs",
-            .partition_label = NULL,
-            .max_files = 5,
-            .format_if_mount_failed = false
-         };
-
-         esp_err_t ret = esp_vfs_spiffs_register(&conf);
-
-         if (ret != ESP_OK) {
-            ESP_LOGE(TAG, "Failed to register SPIFF filesystem");
-            return;
-         }
-
-         lv_obj_t * obj = lv_image_create(lv_screen_active());
-         lv_image_set_src(widget, "A:/spiffs/logo.bin");
-         lv_obj_center(widget);
-      }
-
-- **Build and flash**
-
-   After calling ``idf.py build flash`` the picture should be displayed on the screen.
-
-
-.. note::
-
-   Changes made by ``menuconfig`` are not being tracked in the repository if the ``sdkconfig`` file is added to ``.gitignore``, which is the default for many ESP-IDF projects.
-   To make your configuration permanent, add the following lines to ``sdkconfig.defaults``:
-
-   .. code:: c
-
-      CONFIG_PARTITION_TABLE_CUSTOM=y
-      CONFIG_LV_USE_FS_STDIO=y
-      CONFIG_LV_FS_STDIO_LETTER=65
-      CONFIG_LV_FS_DEFAULT_DRIVER_LETTER=65
-
-Support for Pixel Processing Accelerator
-----------------------------------------
-
-Some ESP32 chip series, like the ESP32-P4 support the Pixel Processing Accelerator hardware (PPA), which is capable of
-speeding-up the filling and image blending operations, this peripheral works with the
-DMA-2D hardware which is responsible to move the input/output buffers into/from the PPA processing engine.
-
-Supported devices
------------------
-
-The Espressif targets that supports the PPA are:
+For a quick start with LVGL and ESP32, the following pre-configured demo projects are available for specific development boards:
 
-- ESP32-P4 series.
+-  `ESP-BOX-3 <https://github.com/lvgl/lv_port_espressif_esp-box-3>`__
+-  `ESP32-S3-LCD-EV-BOARD <https://github.com/lvgl/lv_port_espressif_esp32-s3-lcd-ev-board>`__
+-  `M5Stack-CoreS3 <https://github.com/lvgl/lv_port_espressif_M5Stack_CoreS3>`__
```

Comment:
This is the new location for ESP board supports: https://github.com/lvgl/lv_esp_idf